.changeset/fast-tables-design.md

```diff
@@ -0,0 +1,52 @@
+---
+'xstate': major
+---
+
+- The `execute` option for an interpreted service is no longer inherited by invoked/spawned services. They default to `execute: true`.
```

Comment:
q: could we just remove `execute` altogether? imho it's pretty confusing and avoiding side-effects cant be done in a multiple different ways